networks/litecointestnet/info.json

```diff
@@ -27,7 +27,7 @@
     "networkCode": "litecointestnet",
     "networkName": "Litecoin Testnet",
     "regex": {
-        "address": "^[LM3][a-km-zA-HJ-NP-Z1-9]{26,33}$"
+        "address": "^[LM3Q2mn][a-km-zA-HJ-NP-Z1-9]{26,33}$"
```

Comment:
For legacy addresses the range should be `{25,34}`, same as BTC pretty sure

Reply:
This doesn't include SegWit:
https://stackoverflow.com/a/70983277

This seems like the best option here for me:

`^([LM3Q2mn][a-km-zA-HJ-NP-Z1-9]{25,34}|tltc1[a-z0-9]{39,59})$`

Includes 26-35 length addresses and testnet SegWit.

We should probably also update LTC mainnet with:

`^([LM3][a-km-zA-HJ-NP-Z1-9]{25,34}|ltc1[a-z0-9]{39,59})$`